lib/helpers/create-stormpath-session.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var createSession = require('./create-session');
+var exchangeStormpathToken = require('./exchange-stormpath-token');
+
+/**
+ * Creates a Stormpath JWT, exchanges it for an OAuth token, and stores it in a cookie.
+ *
+ * @param  {Object} account              Account to create a session for.
+ * @param  {Object} req                  Express HTTP request.
+ * @param  {Object} res                  Express HTTP response.
+ * @param  {function} callback           Callback to call once the token cookie(s) has been set.
+ */
+module.exports = function createStormpathToken(account, req, res, callback) {
```

Comment:
The name of this file is `create-stormpath-session.js` and is exported as `createStormpathSession` in `index.js`. Is there any reason why the function is called `createStormpathToken` here?